### PR TITLE
add -x|--nolock arg

### PR DESCRIPTION
**Pull request by pgray**
This adds an arg to pktvisor to not lock memory

### CODE REVIEW

**Review by tlloydaubry — APPROVED** [review]

File: src/pktvisor.c [hunk 1]
```diff
@@ -1214,6 +1216,7 @@ static void __noreturn help(void)
 	     "  -b|--bind-cpu <cpu>            Bind to specific CPU\n"
 	     "  -u|--user <userid>             Drop privileges and change to userid\n"
 	     "  -g|--group <groupid>           Drop privileges and change to groupid\n"
+       "  -x|--nolock                    Don't lock memory\n"
```
Comment (bcavns01) [inline src/pktvisor.c]: That alignment.  You of all people... :disappointed: 

:trollface:

Comment (pgray) [reply src/pktvisor.c]: Weirdly if you look at the full file it's fine.

**Review by elindsey — APPROVED** [review]

**Review by bcavns01 — APPROVED** [review]

File: src/pktvisor.c [hunk 2]
```diff
@@ -1324,6 +1327,9 @@ int main(int argc, char **argv)
 			ctx.gid = strtoul(optarg, NULL, 0);
 			ctx.enforce = true;
 			break;
+    case 'x':
```
Comment (fcelda) [inline src/pktvisor.c]: Tabs.

**Review by fcelda — CHANGES_REQUESTED** [review]
Please, fix the indentation to be consistent with surrounding code. Even though the surrounding code is indented strangely. Otherwise LGTM.

File: src/pktvisor.c (hunk 1 above)
Comment (fcelda) [reply src/pktvisor.c]: @pgray
Screenshot from 2019-08-29 12-31-34

**Review by fcelda — APPROVED** [review]

**Pull request merged**